team-norms/delivery_healthcheck.md

```diff
@@ -0,0 +1,17 @@
+# Delivery Healthcheck
+
+It is the responsibility of the lead on each delivery team to provide an update on the health of their delivery in each week's BizOps showcase.
+
+A Delivery Healthcheck should answer:
+
+## Current iteration goal
```

Comment:
Is "What is your current goal" one of the things to share? If so might want to pull this out of the header to be clearer, and re-head as just "Current iteration".